[sentry-native] typo in patch breaks build for static lib triplet

### Operating system

linux

### Compiler

_No response_

### Steps to reproduce the behavior

```Shell
Install `sentry-native` port with a custom triplet with static linking.
```


### Failure logs

`CMake Error at vcpkg_installed/[custom_triplet]/share/sentry/sentry-config.cmake:37 (find_depenency):
  Unknown CMake command "find_depenency".`


### Additional context

Typo brought in PR #36088.

`find_depenency` should be changed to `find_dependency`

--- a/ports/sentry-native/fix-config-cmake.patch
+++ b/ports/sentry-native/fix-config-cmake.patch
@@ -15,7 +15,7 @@ index 70ce7d3..796d428 100644
  	include("${CMAKE_CURRENT_LIST_DIR}/sentry_crashpad-targets.cmake")
  	if(NOT MSVC AND NOT SENTRY_BUILD_SHARED_LIBS)
 -		find_package(ZLIB REQUIRED)
-+		find_depenency(ZLIB)
++		find_dependency(ZLIB)
  	endif()
  endif()
 

--- a/ports/sentry-native/vcpkg.json
+++ b/ports/sentry-native/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "sentry-native",
   "version": "0.7.0",
+  "port-version": 1,
   "description": "Sentry SDK for C, C++ and native applications.",
   "homepage": "https://sentry.io/",
   "license": "MIT",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2294,7 +2294,7 @@
     },
     "drogon": {
       "baseline": "1.9.2",
-      "port-version": 0 
+      "port-version": 0
     },
     "dstorage": {
       "baseline": "1.2.2",
@@ -7854,7 +7854,7 @@
     },
     "sentry-native": {
       "baseline": "0.7.0",
-      "port-version": 0
+      "port-version": 1
     },
     "septag-dmon": {
       "baseline": "2022-02-08",

--- a/versions/s-/sentry-native.json
+++ b/versions/s-/sentry-native.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "3b95acbd27868522d2e27abe48d9f108d439f523",
+      "version": "0.7.0",
+      "port-version": 1
+    },
+    {
       "git-tree": "5b7467839d13128952c0f944197f8d2fcbc7a763",
       "version": "0.7.0",
       "port-version": 0

